src/themes/default/elements/icon.overrides

```diff
@@ -428,6 +466,10 @@ i.icon.street.view:before { content: "\f21d"; }
 i.icon.hotel:before { content: "\f236"; }
 i.icon.train:before { content: "\f238"; }
 i.icon.subway:before { content: "\f239"; }
+i.icon.map.pin:before { content: "\f276"; }
+i.icon.map.signs:before { content: "\f277"; }
+i.icon.map.outline:before { content: "\f278"; }
+i.icon.map:before { content: "\f279"; }
```

Comment:
Now there's an icon for `map`, so the previous alias to `compass` should be changed.